src/java/org/apache/cassandra/cache/ChunkCache.java

```diff
@@ -62,6 +67,25 @@ public class ChunkCache
     private final static Logger logger = LoggerFactory.getLogger(ChunkCache.class);
 
     public static final int RESERVED_POOL_SPACE_IN_MB = 32;
+    private static final int INITIAL_CAPACITY = Integer.getInteger("cassandra.chunkcache_initialcapacity", 16);
```

Comment:
If it is important for this to be a good estimation, we can calculate it from the cache and chunk sizes (e.g. 512 MB cache / 4-16 kB chunks -> between 32k and 128k initial size).

What does the setting ultimately control that is not automatically adjusted?